xml/Microsoft.Build.Tasks/CreateItem.xml

```diff
@@ -113,12 +113,16 @@
             </summary>
         <value>Additional metadata to attach to the output items.</value>
         <remarks>
-            The fact that this is a string[] makes the following illegal:
-                <CreateItem AdditionalMetadata="TargetPath=@(OutputPathItem)" />
-            The engine fails on this because it doesn't like item lists being concatenated with string
-            constants when the data is being passed into an array parameter.  So the workaround is to 
-            write this in the project file:
-                <CreateItem AdditionalMetadata="@(OutputPathItem-&gt;'TargetPath=%(Identity)')" /></remarks>
+          <format type="text/markdown"><![CDATA[
```

Comment:
Here's an example of where the imported comments from source have better formatting, that we wouldn't get from just importing w/o comments.